src/app/pages/waiting-room-to-car/components/mot-components/alternate-evidence-description/alternate-evidence-description.component.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Component, EventEmitter, Input, Output,
+} from '@angular/core';
+import { UntypedFormControl, UntypedFormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'alternate-evidence-description',
+  templateUrl: './alternate-evidence-description.component.html',
+  styleUrls: ['./alternate-evidence-description.component.scss'],
+})
+export class AlternateEvidenceDescriptionComponent {
+
+  formControl: UntypedFormControl;
+
+  @Input()
+  formGroup: UntypedFormGroup;
+
+  @Output()
+  evidenceDescriptionTestResultChange = new EventEmitter<string>();
+
+  evidenceDescription: string;
+
+  ngOnChanges(): void {
+    if (!this.formControl) {
+      this.formControl = new UntypedFormControl('', [Validators.required]);
+      if (this.formGroup.contains('evidenceDescriptionCtrl')) {
+        this.formControl.patchValue(this.formGroup.controls['evidenceDescriptionCtrl'].value);
+        this.formGroup.setControl('evidenceDescriptionCtrl', this.formControl);
+      } else {
+        this.formGroup.addControl('evidenceDescriptionCtrl', this.formControl);
+      }
+    }
+  }
```

Comment:
Why do we need to check the form contains control here? This is not unlike other fields that we show/hide based upon other values